content/get_started/analog.en.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Getting Started - Analog/Mixed Signal Designers
+weight: 30
+---
+
+As of the fifth cohort, Tiny Tapeout offers support for analog and mixed-signal designs. For those with existing ASIC and HDL experience, this allows for increased flexibility and the creation of new and more intricate designs.
+
+In Tiny Tapeout 6, analog and mixed signal designs were enabled through the addition of an analog multiplexer based on transmission gates. This allows up to 192 designs to share the analog pins between them.
+
+To get started, we recommend consulting our [HDL documentation](/hdl/), which includes templates and information on testing your designs. More information is available on our [technical specifications page](/specs/).
```

Comment:
HDL is for digital designers. It is also partially relevant for analog designers, but most of the info is currently at https://www.tinytapeout.com/specs/analog/